libs/flux/build.go

```diff
@@ -62,6 +62,8 @@ func (t Target) DetermineCargoTarget(logger *zap.Logger) string {
 		return "aarch64-unknown-linux-gnu"
 	case t.OS == "linux" && t.Arch == "arm64" && t.Static:
 		return "aarch64-unknown-linux-musl"
+	case t.OS == "linux" && t.Arch == "s390x":
```

Comment:
Should we add a case for `t.Static` as well? It looks like `s390x-unknown-linux-musl` exists, but it's a Tier 3 target in Rust so I'm not sure how well it'll work.

Reply:
Yes `s390x-unknown-linux-musl` is Tier 3 and therefore the target is not available for download.  I haven't look into it 
too deeply and am not sure how to easily add the target.